docs/source/modules/installation.rst

```diff
@@ -1,28 +1,30 @@
 Installation
 ============
-.. admonition:: Prerequisites
```

Comment:
Have you tried this, but keeping the admonition box? I rather like it.

Reply:
Not yet, but I will try it out now.

Reply:
Looks good, I added it.